client/src/components/BobaCard.jsx

```diff
@@ -0,0 +1,106 @@
+import { React } from "react";
+
+import { Box, Stack, Typography, Divider, useTheme, Link } from "@mui/material";
+
+import BobaCardPath from "../assets/card-milktea_p1.svg";
+
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
+import LanguageIcon from "@mui/icons-material/Language";
+
+const BobaCard = ({
+  name,
+  pronouns,
+  position,
+  linkedInUrl,
+  websiteUrl,
+  icon,
+}) => {
+  const theme = useTheme();
+
+  return (
+    <Box
+      sx={{
+        width: "90%",
+        position: "relative",
+        marginTop: "30px",
+        [theme.breakpoints.down("md")]: {
+          width: "80%",
+        },
+      }}
+    >
+      <Box component="img" src={BobaCardPath} />
+      <Stack
+        alignItems="center"
+        direction="column"
+        sx={{ position: "absolute", width: "100%", top: 0, left: 0 }}
+      >
+        <Box
+          component="img"
+          src={icon}
+          sx={{ width: "40%", marginTop: "48%" }}
+        />
+        <Typography
+          sx={{
+            marginTop: "5%",
+            fontWeight: 600,
+            textAlign: "center",
+            maxWidth: "80%",
+            fontSize: "1.3vw",
```

Comment:
Can we use rem for font sizes so it can respect the base font size? Or is there a typography element that shares any similarities with this that we can piggy back off of?

Reply:
i was initially using rem but I found that using vw yielded the best results in terms of responsiveness. I can change it back to rem but the responsiveness will probably be a bit choppy. Maybe using the [responsive font sizes](https://mui.com/material-ui/customization/theming/#responsivefontsizes-theme-options-theme) that MUI provides could be a useful tool once theme.js is reconfigured

Reply:
https://mui.com/material-ui/customization/theming/#responsivefontsizes-theme-options-theme

Do we have this enabled?

Edit: oh i see you have also linked this. we should use this over manually trying to use diff font sizing, and customize when we have the need

Reply:
responsiveFontSizes() didn't really help much in giving the font behavior that we want, we would still have to set a bunch of breakpoints if we want to use rem. I still think using vw just for the text inside of the cards is the best way to display the cards, it scales very cleanly and doesn't add linebreaks across different window sizes